src/test/ui/issues/issue-30123.stderr

```diff
@@ -3,6 +3,8 @@ error[E0599]: no function or associated item named `new_undirected` found for st
    |
 LL |     let ug = Graph::<i32, i32>::new_undirected();
    |                                 ^^^^^^^^^^^^^^ function or associated item not found in `issue_30123_aux::Graph<i32, i32>`
+   |
+   = note: the function or associated item was found for issue_30123_aux::Graph<N, E, Undirected>
```

Comment:
This would ideally talk about `Graph<i32, i32, Undirected>`, potentially mentioning the `Ty` default of `Directed`. No need to tackle this now.